client/src/components/LoginModal.jsx

```diff
@@ -33,10 +34,19 @@ const LoginModal = () => {
         login(data);
         closeLoginModal()
       } else {
-        console.error('Login failed:', data.message);
+        Modal.warning({
```

Comment:
What's the difference between Modal.warning and Modal.error? Curious to know what changes on UI end from user point of view if we are using Modal.warning versus Modal.error.

Reply:
Modal.warning
<img width="800" alt="modal warning" src="https://github.com/user-attachments/assets/1a97b222-49e5-40b6-ae91-bb000bb406ed">

Modal.error
<img width="855" alt="modal error" src="https://github.com/user-attachments/assets/71169ce2-e820-4573-ae44-bde46c679245">